contracts/main/price-feeders/DelayFathomOraclePriceFeed.sol

```diff
@@ -8,29 +8,27 @@ import "../interfaces/IFathomOracle.sol";
 import "../interfaces/IAccessControlConfig.sol";
 
 contract DelayFathomOraclePriceFeed is PausableUpgradeable, IFathomOraclePriceFeed {
-    struct Feed {
-        uint256 price;
-        uint256 lastUpdateTS;
-    }
-
-    Feed public currentPrice;
+    uint256 public latestPrice;
+    uint256 public delayedPrice;
```

Comment:
Introduced new variable "delayedPrice" to store delayed price and return it on readPrice and peekPrice calls